hack/workspace/go.work

```diff
@@ -0,0 +1,6 @@
+go 1.22.0
```

Comment:
is this going to force us all to use this version of go in order to run `make verify`?

Reply:
we already require that version of go because of https://github.com/openshift/hypershift/blob/main/go.mod